code/api/src/modules/user/mutations.js

```diff
@@ -3,7 +3,7 @@ import { GraphQLString, GraphQLInt } from 'graphql'
 
 // App Imports
 import { UserType } from './types'
-import { create, remove , updateEmailResolver, updateAddressResolver, updateImageResolver} from './resolvers'
+import { create, remove , updateEmailResolver, updateAddressResolver, updateImageResolver, updateUserResolver } from './resolvers'
```

Comment:
We could probably get rid of the resolvers we aren't using (updateEmailResolver, updateAddressResolver, and updateImageResolver) to clean the code up a bit

Reply:
Since we are short on time, maybe we can create an issue to return to clean this up in a future iteration?